tests/test_parsing.py

```diff
@@ -68,6 +68,7 @@ def test_absolute_quality(self):
 class ScanningTest(TestCase):
     def test_scanning(self):
         self.assertRaises(InterfaceError, Cell.all, 'fake-interface')
+        self.assertRaises(InterfaceError, Cell.all, 'fake-interface', True)
```

Comment:
again, can we say `sudo=True` instead of just a naked `True`, that way it's more obvious what's happening.